### ISSUE TITLE
Take over maintenance of the IntelliJ IDEA packages

### ISSUE THREAD

**Issue by AnthonyMastrean** [JetBrains/Chocolatey#7]
I maintain these packages and would love to hand them over :)

https://chocolatey.org/packages/IntelliJIDEA
https://chocolatey.org/packages/intellijidea-community
https://chocolatey.org/packages/intellijidea-ultimate

The source is here

https://github.com/AnthonyMastrean/chocolateypackages/tree/master/packages

I'm about to upload the 2016 editions, but if you want, you can have that pleasure.

**Comment by AnthonyMastrean** [comment]
The JetBrains/Intellij team keeps breaking the URL for old versions. This is causing a tremendous amount of churn. It's one thing not to publish the latest package quickly, it's another when the old version is busted.

**Comment by hhariri** [comment]
@AnthonyMastrean How are they breaking? Could you give me some more details?

**Comment by AnthonyMastrean** [comment]
For example, the download URL for version 15.0 is now 404.

https://download.jetbrains.com/idea/ideaIU-15.0.exe

You can only get version 15.0.5.

https://download.jetbrains.com/idea/ideaIU-15.0.5.exe

This has been happening since I started maintaining the package at version 11.0.

**Comment by hhariri** [comment]
@AnthonyMastrean I'll find it out why. However, I would like to know what policy Chocolately has when a vendor for instance wants to not make available a specific release for whatever the reason.

**Comment by koppor** [comment]
There is no entry in the [FAQ](https://chocolatey.org/faq). There is a description of the [moderation process](https://chocolatey.org/docs/moderation). They have, however a [package triage process](https://chocolatey.org/docs/package-triage-process#the-triage-process), where a site admin may be contacted in special cases. I assume that a withdrawal of a version is such a special cases. I think, that this is a very rare case and vendors "just" take care that their users update frequently.

**Comment by dragon788** [comment]
Yeah, Chocolatey doesn't try to assume too much responsibility for "deprecating" old versions. I will say from a licensing/corporate use stance that having older versions available that were tested to comply with internal policies is valuable, but having most old version links "unpublished" is understandable to avoid users clicking through and getting an older copy on accident. Typically on Chocolatey.org I think you can push a "remove" to a specific version to remove that release, but not sure how often that happens.

**Comment by koppor** [comment]
2016.3 was released today (see https://www.jetbrains.com/idea/download/), so this issue gets important again.

**Comment by dragon788** [comment]
As a maintainer you can also "unlist" a specific version. You can't actually "destroy" the package, but unlisting it makes it not show up in the package list and you can only install that specific version by specifying it exactly.

**Comment by dragon788** [comment]
With [Automatic Packages](https://chocolatey.org/docs/automatic-packages) this should be fairly easy for JetBrains to maintain and keep up with the latest versions.

**Comment by franklinyu** [comment]
Any news? Still concerned about withdrawing particular versions?

**Comment by koppor** [comment]
With JetBrain's [toolbox](https://www.jetbrains.com/toolbox/), all the above packages become (somehow) obsolete. Maybe, someone can package that tool instead?

**Comment by AnthonyMastrean** [comment]
@koppor That sounds good! But, I wonder if the neurotic power user would be happy de-listing all of the other packages? Are there command-line arguments to toolbox for automatically installing certain products or product groups?

**Comment by franklinyu** [comment]
+1 to the Command-Line Interface mentioned by @AnthonyMastrean.

In addition, it would be appreciated if we can install JetBrain Toolbox with Chocolatey. That is easier to maintain than 9 packages.

**Comment by jvtrigueros** [comment]
This repository is now obsolete, does that mean that Jetbrains will not take over the the Toolbox or IntelliJ IDEA Ultimate Chocolatey packages?
